src/app/GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -366,6 +367,17 @@ static void ShowGitRepo(Form? ownerForm, string? workingDir)
                         });
                 }
             }
+
+            static string ReplaceQuotes(string command)
```

Comment:
Very specific usage...
```suggestion
            static string ReplacePrefixQuotes(string command)
```